minio/credentials/credentials.py

```diff
@@ -43,23 +53,24 @@ def __init__(
         self._expiration = expiration
 
     @property
-    def access_key(self):
+    def access_key(self) -> str:
         """Get access key."""
         return self._access_key
 
     @property
-    def secret_key(self):
+    def secret_key(self) -> str:
         """Get secret key."""
         return self._secret_key
 
     @property
-    def session_token(self):
+    def session_token(self) -> str | None:
         """Get session token."""
         return self._session_token
 
-    def is_expired(self):
+    def is_expired(self) -> bool:
         """Check whether this credentials expired or not."""
         return (
             self._expiration < (datetime.utcnow() + timedelta(seconds=10))
-            if self._expiration else False
+            if self._expiration
```

Comment:
Avoid unnecessary diffs

Reply:
done.

have you consider add [black](https://github.com/psf/black) as formatter？

Reply:
We can unless it fixes real issues.

Reply:
it help avoid unnecessary diffs like this...

Reply:
You could disable `black` for `minio-py`

Reply:
I'm using formatter of pycharm, not black here...